charts/tidb-lightning/templates/_helpers.tpl

```diff
@@ -22,6 +22,21 @@ config-file: |-
     {{- if .Values.config }}
 {{ .Values.config | indent 2 }}
     {{- end -}}
+    {{- if and .Values.tlsCluster .Values.tlsCluster.enabled }}
+  [security]
+  ca-path="/etc/lib/lightning-tls/ca.crt"
+  cert-path="/etc/lib/lightning-tls/tls.crt"
+  key-path="/etc/lib/lightning-tls/tls.key"
+    {{- end }}
+    {{- if and .Values.tlsClient .Values.tlsClient.enabled }}
+  [tidb.security]
+  ca-path="/etc/lib/tidb-client-tls/ca.crt"
+  cert-path="/etc/lib/tidb-client-tls/tls.crt"
+  key-path="/etc/lib/tidb-client-tls/tls.key"
+    {{- else }}
+  [tidb]
```

Comment:
What if users configure other parameters in the `[tidb]` section in the `config` field of the [values.yaml](https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-lightning/values.yaml#L82)?
According to the description [here](https://github.com/pingcap/tidb-lightning/blob/master/tidb-lightning.toml#L204), it seems that the default empty value can meet our requirement.

Reply:
if the user set `tlsCluster.enabled: true`, then the default empty value will become `cluster`.

Reply:
https://github.com/pingcap/tidb-lightning/blob/56bc32daad19b9dff10104c55300292de959fde3/lightning/config/config.go#L552-L571

Reply:
OK, got it, considering the other configuration parameters in the `[tidb]` section, I think we can request users to set `tls: false` manually in the doc for this case.

Reply:
removed `[tidb]` section in https://github.com/pingcap/tidb-operator/pull/3598/commits/68200caf75269b0ce2e84b69ce41d22a28e8caf7.